### DISCUSSION TITLE
Database additions...

### DISCUSSION THREAD

**Discussion by Fxdli** [Itz-Hyperz/firewallgg#24] [Q&A] [answered]
What are the requirements for adding a database? Like, can anyone do it?

**Comment by Itz-Hyperz** [comment, marked as answer]
We only support databases that are on the "same side" as us. We are looking to store and connect databases that prevent bad users within Discord. So if your database does not meet those same standards, odds are it will not be accepted...

**Reply by Fxdli** [reply]
okay, thank you!
